README.md

```diff
@@ -9,6 +9,7 @@ Of course the tested routers can be used for any kind of HTTP request → handle
 
 #### Tested routers & frameworks:
 
+ * [Baa](https://github.com/go-baa/baa)
```

Comment:
Please don't edit the README. It lists the routers tested in that round. Baa will be included in the next round :wink: